tests/jeos/firstrun.pm

```diff
@@ -294,9 +294,6 @@ sub run {
         } else {
             send_key 'n';
         }
-    }
-
-    if (is_tumbleweed || is_sle_micro('>6.0') || is_microos) {
```

Comment:
This should not be removed but instead `is_sle('>=16')` should be added here.

Reply:
Yes and no. The only code that was inside this condition is `create_user_in_ui();` which is now part of the previous condition (they were the same condition but using unless and if, I unified them)

Reply:
Ok, that makes sense. Let's see.